docs-client/src/containers/MethodPage/DebugPage.tsx

```diff
@@ -203,6 +227,10 @@ const DebugPage: React.FunctionComponent<Props> = ({
         )?.pathMapping || '';
     }
 
+    if (urlPath.startsWith('/')) {
+      urlPath = parseServerRootPath(docServiceRoute) + urlPath;
```

Comment:
Question) Should `Copy as Curl` also use the root path?

Reply:
I think we should have the same behavior there as well, so yes.